src/gui/optionsdialog.cpp

```diff
@@ -399,7 +399,9 @@ OptionsDialog::OptionsDialog(QWidget *parent)
             , tr("%C: Number of files")
             , tr("%Z: Torrent size (bytes)"))
         .arg(tr("%T: Current tracker")
-            , tr("%I: Info hash")
+            , tr("%I: Info hash v1")
+            , tr("%J: Info hash v2")
```

Comment:
Now since they are not always available, you should add note of what is expected when not available.